src/interflop-stdlib/automake-expand

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
```

Comment:
When and how is automake-expand used ? I'm not sure to understand.

Reply:
This script was used to populate the interflop-config script, but I ultimately decided to remove it since it would add an extra layer of complexity that is not necessary. The idea was to have something similar to llvm-config, but I think it's overkill.